### PR TITLE
fix: Update root docs for removing Build reference.

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rhuss** [comment]
@joaedwar we have to use that fix, as the documentation is auto generated. As the description is also used as textual output for `kn --help`, we can't use markdown links here, as this doesnt render nicely on the console. But I added a bullet list and removed teh "build" reference.

File: pkg/kn/core/root.go [hunk 1]
```diff
@@ -49,9 +49,8 @@ func NewKnCommand(params ...commands.KnParams) *cobra.Command {
 		Short: "Knative client",
 		Long: `Manage your Knative building blocks:
 
-Serving: Manage your services and release new software to them.
-Build: Create builds and keep track of their results.
```
Comment (navidshaikh) [inline pkg/kn/core/root.go]: Lets remove it from https://github.com/knative/client/blob/master/pkg/kn/core/root.go#L53 , else it will come back.

Comment (rhuss) [reply pkg/kn/core/root.go]: It is already removed, isn't it ? (you are refering to a deleted line)

Comment (navidshaikh) [reply pkg/kn/core/root.go]: damn!

**Comment by rhuss** [comment]
> @rhuss: you cannot LGTM your own PR.

thought it was small enough to fail. But yes, have to wait then.

**Comment by rhuss** [comment]
@sixolet @cppforlife @maximilien Can we have a quick /lgtm for this mini doc fix ? It slipped in and now every new PR get the bogus generated docs as part of the PR. 

thanks

**Comment by maximilien** [comment]
@rhuss it's likely that the unresolved conflict  in `docs/cmd/kn.md` is preventing the merge.

**Comment by rhuss** [comment]
@cppforlife @maximilien conflict resolved, PTAL for an /lgtm

**Comment by rhuss** [comment]
/retest

Flakiness in smoke test should be fixed with #250

**Pull request merged**